src/middlewares/upload_logo.ts

```diff
@@ -29,16 +30,20 @@ import {
   tryCatch
 } from "fp-ts/lib/TaskEither";
 
+import { Option } from "fp-ts/lib/Option";
+
 import { ErrorResponses, notificationApiClient } from "../controllers/services";
 
+import { SubscriptionContract } from "azure-arm-apimanagement/lib/models";
 import { fromNullable, toError } from "fp-ts/lib/Either";
+import { NonEmptyString } from "italia-ts-commons/lib/strings";
 import { ServiceId } from "../../generated/api/ServiceId";
 
 export const getApimUserTask = (
   apiClient: ApiManagementClient,
   authenticatedUser: AdUser
 ): TaskEither<ErrorResponses, IExtendedUserContract> =>
-  tryCatch(
+  tryCatch<ErrorResponses, Option<IExtendedUserContract>>(
```

Comment:
Generics is optional here
```suggestion
  tryCatch(
```